packages/cli/src/lib/bundler/server.ts

```diff
@@ -139,58 +160,102 @@ export async function serveBundle(options: ServeOptions) {
     additionalEntryPoints: detectedModulesEntryPoint,
   });
 
-  const compiler = webpack(config);
-
-  server = new WebpackDevServer(
-    {
-      hot: !process.env.CI,
-      devMiddleware: {
-        publicPath: config.output?.publicPath as string,
-        stats: 'errors-warnings',
+  console.log(paths.targetHtml);
+  if (process.env.EXPERIMENTAL_VITE) {
+    server = await vite.createServer({
+      define: {
+        global: 'globalThis',
```

Comment:
literally that string? i presume it's an enum so vite does something sensible internally

Reply:
yeah `globalThis === window` in the browser. but it's also defined in node land too. Gonna try and change this to `window` to see if it still works to clear maybe some confusion.